knimin/tests/test_barcode_util.py

```diff
@@ -307,23 +318,27 @@ def test_get_ag_details_SJ(self):
         del ag_details['overloaded']
         del ag_details['other']
         del ag_details['deposited']
-        self.assertEqual(ag_details,
-                         {'login_user': 'REMOVED',
-                          'environment_sampled': '', 'withdrawn': '',
-                          'ag_kit_id': 'd8592c74-7ddb-2135-e040-8a80115d6401',
-                          'overloaded_checked': '',
-                          'participant_name': 'REMOVED-0',
-                          'ag_kit_barcode_id':
-                          'd8592c74-7ddc-2135-e040-8a80115d6401',
-                          'sample_date': date(2013, 4, 18),
-                          'other_checked': '', 'status': 'Received',
-                          'refunded': '', 'other_text': 'REMOVED',
-                          'barcode': '000001018', 'moldy_checked': '',
-                          'date_of_last_email': '', 'site_sampled': 'Stool',
-                          'email_type': '1', 'name': 'REMOVED',
-                          'sample_time': time(6, 50),
-                          'notes': 'REMOVED',
-                          'email': 'REMOVED'})
+        exp = {
+               # 'login_user': 'REMOVED',
+               'environment_sampled': '', 'withdrawn': '',
+               'ag_kit_id': 'd8592c74-7ddb-2135-e040-8a80115d6401',
+               'overloaded_checked': '',
+               # 'participant_name': 'REMOVED-0',
+               'ag_kit_barcode_id':
+               'd8592c74-7ddc-2135-e040-8a80115d6401',
+               'sample_date': date(2013, 4, 18),
+               'other_checked': '', 'status': 'Received',
+               'refunded': '', 'other_text': 'REMOVED',
+               'barcode': '000001018', 'moldy_checked': '',
+               'date_of_last_email': '', 'site_sampled': 'Stool',
+               # 'email_type': '1',
+               # 'name': 'REMOVED',
+               'sample_time': time(6, 50),
+               # 'notes': 'REMOVED',
+               # 'email': 'REMOVED'
+              }
+        # only look at those fields, that are not subject to scrubbing
+        self.assertEqual({k: ag_details[k] for k in exp}, exp)
```

Comment:
Could you add a check for the keys? In case, that in the future we add more elements, we know that we are actually testing those ...